package.json

```diff
@@ -19,16 +19,16 @@
   "homepage": "https://github.com/open-telemetry/assign-reviewers-action#readme",
   "devDependencies": {
     "@types/js-yaml": "^4.0.2",
-    "@types/node": "^16.4.12",
-    "@vercel/ncc": "^0.29.0",
+    "@types/node": "^16.18.23",
+    "@vercel/ncc": "^0.29.2",
     "rimraf": "^3.0.2",
-    "typescript": "^4.3.5"
+    "typescript": "^4.9.5"
   },
   "dependencies": {
     "@actions/core": "^1.4.0",
     "@actions/github": "^5.0.0",
-    "fp-ts": "^2.11.1",
-    "io-ts": "^2.2.16",
+    "fp-ts": "^2.13.1",
+    "io-ts": "^2.2.20",
```

Comment:
Some of the dependency updates aren't specifically needed but I think they're all a good idea